feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt

```diff
@@ -86,7 +88,8 @@ internal fun HomeScreenUI(
             sheetControls = {
                 FeedingPointActionButton(
                     alpha = buttonAlpha,
-                    onClick = { onScreenEvent(ShowWillFeedDialog) }
+                    onClick = { onScreenEvent(ShowWillFeedDialog) },
+                    enabled = state.currentFeedingPoint?.animalStatus != AnimalState.GREEN
```

Comment:
Please, make `== AnimalState.RED` instead. Later we will have yellow state, which will disable the button as well